style.css

```diff
@@ -198,7 +198,7 @@ header ul li {
   line-height: 16px;
```

Comment:
[OPTIONAL] I suggest your page since it does not look like the overall design. The social media icons `twitter, medium, etc` are not showing, neither is the background image for the headline section.
This is your page:
<img width="1676" alt="Screenshot 2023-01-19 at 09 41 04" src="https://user-images.githubusercontent.com/48546992/213373908-9659a06a-9634-4a9d-a0a0-335bc2082866.png">
This is how your page should look
<img width="381" alt="Screenshot 2023-01-19 at 09 40 33" src="https://user-images.githubusercontent.com/48546992/213374083-6240210e-c2ee-4590-aa41-23e8e75f31c1.png">
I only marked this as optional, since it had already been approved,  but remember it's a critical part of your project and your project should be a copy of the figma template, to show that given a particular design you are able to replicate it. Please do not merge before fixing it. It will be checked for the desktop version.